### ISSUE TITLE
Problem with pull secret

### ISSUE THREAD

**Issue by vsychov** [argoproj-labs/argocd-image-updater#195]
**Describe the bug**
Hi, I found very minor bug, when in `registries.conf`, one of registry have `api_url` that end with `/`, e.g. `https://example.com/`, and pull secret that have `example.com` as server, it will not be used.

**To Reproduce**
```
- name: Registry
  api_url: https://example.com/
  prefix: example.com
  ping: no
  credentials: pullsecret:default/secret-name
```

Looks like problem here - https://github.com/argoproj-labs/argocd-image-updater/blob/37433e46a136f37ed130b7914dd50be26b929515/pkg/image/credentials.go#L223-L230

`/` in the end should be removed.

**Expected behavior**
Secret found.

**Additional context**
-

**Version**
v0.9.2

**Logs**
Please paste any relevant logs here

**Comment by jannfis** [comment]
Nice catch, @vsychov! Would you also like to send a PR to fix it?

**Comment by vsychov** [comment]
@jannfis , fixed by #197
